vllm/lora/punica_wrapper/punica_cpu.py

```diff
@@ -0,0 +1,14 @@
+from typing import final
+
+from .punica_gpu import PunicaWrapperGPU
+
+
+@final
+class PunicaWrapperCPU(PunicaWrapperGPU):
+    """
+    PunicaWrapperCPU is designed to manage and provide metadata for the punica 
+    kernel. The main function is to maintain the state information for 
+    Multi-LoRA, and to provide the interface for the pytorch punica ops.
+    It uses the punica ops in the same manner as the GPU implementation.
+    """
+    pass
```

Comment:
We cannot inherit from `PunicaWrapperGPU` . We are doing Triton kernel optimization, and things in this class will be modified.

Reply:
I thought that the PunicaWrapperGPU class just delegates to the triton kernels, and since we're not using triton here, we'd end up using the pytorch ops?

Reply:
`PunicaWrapperGPU` can only use Triton ops, and different pllatform `PunicaWrapper` should be decoupled. Changes to one shouldn't affect the other. For example, for performance reasons, we might add a fused QKV LoRA triton, and in that case, the Triton ops and Torch ops would no longer have a one-to-one correspondence